util/fipstools/acvp/acvptool/subprocess/rsa.go

```diff
@@ -190,16 +191,28 @@ func processSigGen(vectorSet []byte, m Transactable) (interface{}, error) {
 				return nil, err
 			}
 
+			n := results[0]
+			e := results[1]
+			sig := results[2]
+
 			if len(response.N) == 0 {
-				response.N = hex.EncodeToString(results[0])
-				response.E = hex.EncodeToString(results[1])
-			} else if response.N != hex.EncodeToString(results[0]) {
+				response.N = hex.EncodeToString(n)
+				response.E = hex.EncodeToString(e)
+			} else if response.N != hex.EncodeToString(n) {
 				return nil, fmt.Errorf("module wrapper returned different RSA keys for the same SigGen configuration")
 			}
 
+			// Ask the subprocess to verify the generated signature for this test case.
+			ver_results, ver_err := m.Transact(ver_operation, 1, n, e, msg, sig)
+			if ver_err != nil {
+				return nil, ver_err
+			}
+			if len(ver_results[0]) != 1 || ver_results[0][0] != 1 {
```

Comment:
For my own education: why is this condition different from https://github.com/awslabs/aws-lc/pull/264/files#diff-37b3564b00627cf930702616136708842e9c4c08bdce38f9e3e91fd3b8eb1ab9R184?

Reply:
The return from modulewrapper.cc is diff.